she_codes_news/news/models.py

```diff
@@ -1,8 +1,27 @@
+from django.contrib.auth import get_user_model
 from django.db import models
 
+USER = get_user_model()
 
+class Category(models.Model):
+    name = models.CharField(max_length=64, unique=True)
+    class Meta:
+        verbose_name_plural = "categories"
+
+    def __str__(self):
+        return self.name
 class NewsStory(models.Model):
+    class Meta: 
+        ordering = ["-pub_date"]
```

Comment:
I tend to avoid adding ordering to my models, i think this should be done at the query level